app/Models/Feed.php

```diff
@@ -71,9 +70,6 @@ public function hubUrl() {
 	public function category() {
 		return $this->category;
 	}
-	public function entries() {
```

Comment:
On demand, instead of pre-loading a copy in memory

Reply:
Did you check that it wasn't used anywhere else?

Reply:
I could not find any other use. I have not checked extensions though, and I could introduce a compatibility method returning the entries anyway, but on demand

Reply:
If it's not too complicated, it would be better… and with a warning message in logs to say it's deprecated

Reply:
Example of use in an extension
https://github.com/kevinpapst/freshrss-youtube/blob/06f346789893adbfed10fa00ecf01f6ae14b6282/xExtension-YouTube/extension.php#L251

Reply:
Back-compatibility done by https://github.com/FreshRSS/FreshRSS/pull/2972/commits/578444a904da6aedfd9ad07ff7d84aab1d0af650
I have checked a few extensions without finding any use of it